Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -104,7 +104,7 @@ protected function configure()
                     new InputOption('config-file', '', InputOption::VALUE_OPTIONAL, 'The path to a .php_cs file ', null),
                     new InputOption('dry-run', '', InputOption::VALUE_NONE, 'Only shows which files would have been modified'),
                     new InputOption('level', '', InputOption::VALUE_REQUIRED, 'The level of fixes (can be psr0, psr1, psr2, or symfony (formerly all))', null),
-                    new InputOption('using-cache', '', InputOption::VALUE_REQUIRED, 'Does cache should be used (can be yes or no)', null),
+                    new InputOption('using-cache', '', InputOption::VALUE_OPTIONAL, 'Set cache should be used (can be yes or no)', 'yes'),
```

Comment:
Please see https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/master#caching .

The main configuration comes from Config class or it's configured instance returned by .php_cs file.
Now you set cache parameter to `yes` by default, therefore ignore someone configuration from config/config file.

This can't be done that way.